plugins/pulp_rpm/plugins/distributors/iso_distributor/distributor.py

```diff
@@ -61,18 +61,19 @@ def publish_repo(self, transfer_repo, publish_conduit, config):
         repo = transfer_repo.repo_obj
         return super(ISODistributor, self).publish_repo(repo, publish_conduit, config)
 
-    def unpublish_repo(self, repo, config):
+    def unpublish_repo(self, transfer_repo, config):
```

Comment:
If we change ``FileDistributor``, we'll have to change it's parent class, and then change everything. I'm fine with leaving this as you have it to be consistent within this file.